packer/provision.sh

```diff
@@ -46,10 +44,13 @@ rm /tmp/echidna.tar.gz
 echo "Downloading medusa..."
 sudo apt-get update
 sudo apt-get install -y unzip
-curl -fsSL https://github.com/crytic/medusa/releases/download/v0.1.0/medusa-linux-x64.zip -o medusa.zip
-unzip medusa.zip
+curl -fsSL https://github.com/crytic/medusa/releases/download/v0.1.3/medusa-linux-x64.tar.gz -o /tmp/medusa.tar.gz
+echo "Extracting medusa..."
+tar -xzf /tmp/medusa.tar.gz -C /tmp
+echo "Installing medusa..."
 chmod +x medusa
 sudo mv medusa /usr/local/bin
```

Comment:
```suggestion
chmod +x /tmp/medusa
sudo mv /tmp/medusa /usr/local/bin
```

Just so it's a bit more clear, I don't see a "cd /tmp" anywhere here 🤔